components/input/TextArea.tsx

```diff
@@ -13,6 +13,7 @@ export interface TextAreaProps extends RcTextAreaProps {
   allowClear?: boolean;
   bordered?: boolean;
   showCount?: boolean;
+  countFormatter?: (count: number, maxLength?: number) => string;
```

Comment:
```
showCount={{
  render: ({ count?: number, maxLength?: number }) => React.ReactNode;
}}
```

Reply:
1. Why use the `count?`, the `valueLength` must be defined.
https://github.com/ant-design/ant-design/blob/97ea75c6a14693f167cb7cf5128acaf10d215491/components/input/TextArea.tsx#L119
2. Why return a `React.ReactNode`, the `data-count` only receive a string.
https://github.com/ant-design/ant-design/blob/97ea75c6a14693f167cb7cf5128acaf10d215491/components/input/TextArea.tsx#L133

Reply:
> Why return a React.ReactNode, the data-count only receive a string.

You are right, current implementation allow string only.

Reply:
```
showCount={{
  formatter: ({ count: number, maxLength?: number }) => string;
}}
```

Reply:
Thanks, I'll update later.